lightcurve/templates/lightcurve.html

```diff
@@ -18,15 +18,22 @@
 .image-container {
   margin: 2em 0;
 }
+
 </style>
 {% endblock %}
 
 {% block heading %}
-  Light Curve <span>{{lightcurve.name}}</span>
-  <a href="{% url "upload_image" %}/{{lightcurve.id}}" class="btn btn-primary pull-right">Add New Images</a>
+  <form class="js-edit-name-form" onsubmit="saveLightcurveChanges({{lightcurve.id}}); return false;" method="post">
+    Light Curve
+    <span class="lightcurve-name">{{lightcurve.name}}</span>
+    <input type='name' class='input_name' name='input_name' style="display:none" value='{{lightcurve.name}}'>
+    <button type="button" class="btn btn-xs btn-default js-edit-name" style="display:inline-block">Edit Name</button>
+    <button type="submit" class="btn btn-sm btn-primary submit-name" style="display:none">Save</button>
```

Comment:
Seems like this should be `js-submit-name`, because it is used only by js.

Reply:
Done.